src/Test.UnitTests.Sarif.Driver/Sdk/SkimmerBaseTestResources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="NotApplicable_InvalidMetadata" xml:space="preserve">
+    <value>'{0}' was not evaluated for check '{1}' as the analysis is not relevant based on observed metadata: {2}.</value>
```

Comment:
>as [](start = 51, length = 2)

"because"? Simpler for ESL readers. I like "as" myself, but you know me, I'm Fiftys Man.